.travis.yml

```diff
@@ -1,19 +1,36 @@
 language: python
 dist: trusty
 sudo: required
-matrix:
-    include:
-        - python: 2.7
-          env: PYENV=py27
-        - python: 3.3
-          env: PYENV=py33
-        - python: 3.5
-          env: PYENV=py35
+python:
+    - 2.7
+    - 3.5
```

Comment:
I prefer to test with an older version of Python 3 (3.3 or 3.4), since HPC environments do not always have recent versions available.

Reply:
Ah, yes, of course. I'm not modifying the `.travis.yml` file now, so it'll be as you left it.